packages/rich-text/src/is-collapsed.ts

```diff
@@ -1,17 +1,23 @@
-/** @typedef {import('./create').RichTextValue} RichTextValue */
+/**
+ * Internal dependencies
+ */
+import type { RichTextValue } from './types';
 
 /**
  * Check if the selection of a Rich Text value is collapsed or not. Collapsed
  * means that no characters are selected, but there is a caret present. If there
  * is no selection, `undefined` will be returned. This is similar to
  * `window.getSelection().isCollapsed()`.
  *
- * @param {RichTextValue} value The rich text value to check.
- *
- * @return {boolean|undefined} True if the selection is collapsed, false if not,
- *                             undefined if there is no selection.
+ * @param props       RichTextValue The rich text value to check.
+ * @param props.start
+ * @param props.end
+ * @return True if the selection is collapsed, false if not, undefined if there is no selection.
```

Comment:
Note: this runs into the same issue I [encountered here](https://github.com/WordPress/gutenberg/pull/49649#discussion_r1166017297), where the documentation generator won't find the parameters. I attempt a fix in docgen here: #49812